### PR TITLE
Update Page 1.html

### PR DESCRIPTION
**Pull request by Trucanhnguyen**
Please take a look at my code. Thanks.

### CODE REVIEW

File: Page 1.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h1>Global Issues</h1>
   <img src="images/congo.jpg" alt="hunger" width="600" height="350">
 
   <div>
-      <p>3.Syria</p>
+      <p>3.Syria.</p>
       <p>5.7 millions people</p>
       <a href="index.html">More Details</a>
```
Comment (fabioaarito) [inline Page 1.html:44]: Hi @Trucanhnguyen! I think you don't have any index.html in your repository. maybe you want here to jump to page2.html

File: Page 1.html [hunk 2]
```diff
@@ -39,7 +39,7 @@ <h1>Global Issues</h1>
   <img src="images/congo.jpg" alt="hunger" width="600" height="350">
```
Comment (fabioaarito) [inline Page 1.html:39]: I didn't see any images directory in your repository.

File: Page 1.html (hunk 1 above)
Comment (Trucanhnguyen) [reply Page 1.html:44]: Hi, my index.html is my page 2. I rename it.

File: Page 1.html (hunk 2 above)
Comment (Trucanhnguyen) [reply Page 1.html:39]: Hi @fabioaarito lets me upload it. Thanks.

**Pull request merged**